pkgs/top-level/all-packages.nix

```diff
@@ -11522,9 +11522,13 @@ with pkgs;
 
   qovery-cli = callPackage ../tools/admin/qovery-cli { };
 
-  qownnotes = darwin.apple_sdk_11_0.callPackage ../applications/office/qownnotes {
+  qownnotes-qt5 = if stdenv.isDarwin then darwin.apple_sdk_11_0.callPackage ../applications/office/qownnotes {
     inherit (libsForQt5) qmake qtbase qtdeclarative qtsvg qttools qtwayland qtwebsockets qtx11extras qtxmlpatterns wrapQtAppsHook;
-  };
+  } else libsForQt5.callPackage ../applications/office/qownnotes { };
+  qownnotes-qt6 = if stdenv.isDarwin then darwin.apple_sdk_11_0.callPackage ../applications/office/qownnotes {
+    inherit (qt6Packages) qmake qtbase qtdeclarative qtsvg qttools qtwayland qtwebsockets qt5compat wrapQtAppsHook;
+  } else qt6Packages.callPackage ../applications/office/qownnotes { };
```

Comment:
```suggestion
  qownnotes-qt6 = qt6Packages.callPackage ../applications/office/qownnotes {
    stdenv = if stdenv.isDarwin then darwin.apple_sdk_11_0.stdenv else stdenv;
  };
```

Reply:
The `stdenv` override is needed for `libsForQt5.callPackage`, but not necessarily for `qt6Packages.callPackage`, I guess.